app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java

```diff
@@ -2038,7 +2038,10 @@ private void setupBrightness() {
             restoreDefaultBrightness();
         } else {
             // Do not restore if user has disabled brightness gesture
-            if (!PlayerHelper.isBrightnessGestureEnabled(activity)) {
+            if (PlayerHelper.getActionForRightGestureSide(activity)
+                    .equals(getString(R.string.brightness_control_key))
+                    || PlayerHelper.getActionForLeftGestureSide(activity)
+                    .equals(getString(R.string.brightness_control_key))) {
```

Comment:
I think you have inverted the conditions here. The brightness should be **not** restored only if **both** gestures are **not** brightness.
```suggestion
            if (!PlayerHelper.getActionForRightGestureSide(activity)
                    .equals(getString(R.string.brightness_control_key))
                    && !PlayerHelper.getActionForLeftGestureSide(activity)
                    .equals(getString(R.string.brightness_control_key))) {
```